config/ext/rails.yml

```diff
@@ -243,6 +283,20 @@ Rails/EnvironmentComparison:
   Enabled: true
   VersionAdded: '0.52'
 
+Rails/EnvironmentVariableAccess:
+  Description: 'Do not access `ENV` directly after initialization.'
+  # TODO: Set to `pending` status in RuboCop Rails 2 series when migration doc will be written.
```

Comment:
When I understand this comment correctly from the Rubcop team, we can safely remove it. It's confusing to have this as TODO in our code base.